### PR TITLE
Update HOWTO.md

### PR DESCRIPTION
**Pull request by karamellpelle**
Elaborating some details

#### Description

I added some small clarifications/explanations about how to use _provider-pkcs11_. These explanations would have saved me from some hours of problems while setting up _provider_pkcs11_ on my system. 

I tried to write these additions concise and in similar wording.

#### Checklist

- [ ] Code modified for feature
- [ ] Test suite updated with functionality tests
- [ ] Test suite updated with negative tests
- [x] Documentation updated

#### Reviewer's checklist:

- [ ] Any issues marked for closing are addressed
- [ ] There is a test suite reasonably covering new functionality or modifications
- [ ] This feature/change has adequate documentation added
- [ ] Code conform to coding style that today cannot yet be enforced via the check style test
- [ ] Commits have short titles and sensible commit messages
- [ ] Coverity Scan has run if needed (code PR) and no new defects were found

### CODE REVIEW

File: HOWTO.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ pkcs11-module-init-args = <initialization string here>
 
 Once the section is properly constructed add the following statement to the
 provider section. If a provider section does not exist make sure to create one
-with all the needed providers (at least the default provider will be needed):
+with all the needed providers (at least the default provider will be needed - 
+remember to activate it, otherwise the _openssl_ command will behave strange):
```
Comment (simo5) [inline HOWTO.md:33]: `will behave in a strange way`,
or
`will not behave correctly`

**Review by simo5 — CHANGES_REQUESTED** [review]
Thank you, it is a welcome addition to our docs. Just one minor change required.

**Review by karamellpelle — COMMENTED** [review]
Improved sentence

**Comment by simo5** [comment]
It would be nice if you could squash the two commits, if you can't do that let me know and I will deal with it.

**Comment by karamellpelle** [comment]
> It would be nice if you could squash the two commits, if you can't do that let me know and I will deal with it.

Yes, please do that (and modify the commit even more if you need to). I tried to squash (`git rebase -i HEAD~2` + and push back to my fork repository), but I wasn't able to make it work. I've never done _squashing_ before, and I have little experience with pull request, but it is fun to learn new workflows so I can be better at contributing to the open source community. 

Thanks!

**Comment by simo5** [comment]
I forced pushed the sqashed commits to your main branch.
You probably want to git pull --rebase locally, or something

**Review by simo5 — APPROVED** [review]

**Pull request merged**